packages/opentelemetry-metrics/README.md

```diff
@@ -76,18 +76,75 @@ boundCounter.add(Math.random() > 0.5 ? 1 : -1);
 
 ### Value Observer
 
-Choose this kind of metric when only last value is important without worry about aggregation
+Choose this kind of metric when only last value is important without worry about aggregation.
+The callback can be sync or async.
 
 ```js
 const { MeterProvider } = require('@opentelemetry/metrics');
 
 const meter = new MeterProvider().getMeter('your-meter-name');
 
-meter.createValueObserver('cpu_core_usage', {
-  description: 'Example of a sync observer with callback',
+
+// async callback - for operation that needs to wait for value
+meter.createValueObserver('your_metric_name', {
+  description: 'Example of a async observer with callback',
+}, async (observerResult) => {
+  const value = getAsyncValue();
+  observerResult.observe(value, { label: '1' });
```

Comment:
Looks like this should be `observerResult.observe(await value, { label: '1' });`

Reply:
Think moving the await to `const value = await getAsyncValue();` is more idiomatic